developer_manual/core/ocs-share-api.rst

```diff
@@ -129,3 +130,64 @@ Statuscodes:
 * 400 - wrong or no update parameter given
 * 403 - public upload disabled by the admin
 * 404 - couldn't update share
+
+
+Federated Cloud Shares
+======================
+
+Create a new Federated Cloud Share
+----------------------------------
+
+Creating a federated cloud share can be done via the local share endpoint, using
+(int) 6 as a shareType and the `Federated Cloud ID <https://owncloud.org/federation/>`_
+as shareWith. See `Create a new Share`_ for more information.
+
+
+List pending Federated Cloud Shares
+-----------------------------------
+
+Get all pending federated cloud shares from the user.
+
+* Syntax: /remote_shares
+* Method: GET
+
+* Result: XML with all pending federated cloud shares
+
+Statuscodes:
+
+* 100 - successful
+* 404 - couldn't fetch shares
```

Comment:
What do we get when federated shares are disabled (incoming or outgoing, depending on the API call) ?